apertium-tur-uzb.tur-uzb.lrx

```diff
@@ -1,25 +1,139 @@
 <rules>
 
-<rule weight="1"> 
-    <match lemma="boyunca" tags="post">
-    <select lemma="bo'ylab" tags="post"/>
-    </match>
-</rule>
-
-<rule weight="2"> 
-    <match lemma="*" tags="num"/>
-    <match lemma="*" tags="n.*"/>
-    <match lemma="boyunca" tags="post*">
+
+  <rule weight ="1">  <!-- default -->
+  <match lemma="saklan" tags="v.*">
+    <select lemma="bekin" tags="v.*"/>
+  </match>
+  </rule>
+
+
+  <rule weight ="1">  <!-- default -->
+  <match lemma="küçük" tags="adj">
+    <select lemma="kichik" tags="adj"/>
+  </match>
+  </rule>
+
+  <rule weight ="1">  <!-- default -->
+  <match lemma="dışarı">
+    <select lemma="tashqari"/>
+  </match>
+  </rule>
+
+  <rule weight ="1">  <!-- default -->
+  <match lemma="güzel" tags="adj">
+    <select lemma="goʻzal" tags="adj"/>
+  </match>
+  </rule>
+  <rule weight ="2">
+  <match lemma="çok"/> <!-- if güzel comes after çok, it means "very good"(mostly) (Ex: Hava bugün çok güzel) -->
+  <match lemma="güzel" tags="adj">
+    <select lemma="ajoyib" tags="adj"/>
+  </match>
+  </rule>
+
+
+  <rule weight ="1">  <!-- default -->
+  <match lemma="boyunca" tags="post">
+    <select lemma="boʻylab" tags="post"/>
+  </match>
+  </rule>
+  <rule weight ="2"> <!-- if boyunca comes after a number followed by a noun (Ex: iki gün boyunca) --> <!-- Currently not working -->
+  <match lemma="*" tags="num" /> <!-- just any number? -->
+  <match lemma="*" tags="n.*" /> <!-- just any noun? -->
+  <match lemma="boyunca" tags="post">
     <select lemma="davomida" tags="post"/>
-    </match>
-</rule>
+  </match>
+  </rule>
+
+
+  <rule weight ="1">  <!-- default -->
+  <match lemma="ardından" tags="post">
+    <select lemma="ortidan" tags="post"/>
+  </match>
+  </rule>
+
+
+  <rule weight ="1">  <!-- default -->
+  <match lemma="çalış" tags="v.tv.*">
+    <select lemma="ishla" tags="v.tv.*"/>
+  </match>
+  </rule>
+  <rule weight ="2">  <!-- default -->
+  <match lemma="*" tags="v.*.dat" /> <!-- if konuş comes after dative gerund (Ex: tercüme etmeye çalıştı) -->
+  <match lemma="çalış" tags="v.tv.*">
+    <select lemma="harakat qil" tags="v.tv.*"/>
+  </match>
+  </rule>
+
+
+  <rule weight ="1">  <!-- default -->
+  <match lemma="konuş" tags="v.tv">
+    <select lemma="gapir" tags="v.tv"/>
+  </match>
+  </rule>
+  <rule weight ="2">
+  <or>
+	<match lemma="*" tags="*.ins" /> <!-- if konuş comes after noun/pronoun+Instrumental(comitative) (Ex: Köpekle konuştum)  -->
+	<match lemma="ile" tags="post" /> <!-- if konuş comes after ile (Ex: Ay ile konuştum)  -->
+  </or>
+  <match lemma="konuş" tags="v.tv.*">
+    <select lemma="gaplash" tags="v.tv.*"/>
+  </match>
+  </rule>
```

Comment:
is that means with animals you are using "gaplash"?
In Turkish, we can say  "annemle konuştum"

Reply:
@sevilaybayatli, this rule matches instrumental forms (like "annemle") or forms with "ile" after them and translates konuş as gaplash.  The default rule is for when there's no instrumental or "ile", in which case it's just translated "gapir".  This seems correct to me.

For example, you say "Turkcha gapira olaman" for "Türkçe konuşabilirim", and you wouldn't use "gaplash" here.  It's the distinction between "speak" (gapir) and "converse with" (gaplash) in English (though you can use "speak" with both).

Reply:
yeah,  my question was not to understand if the rule correct or not, but I want to understand the words, because in -tki, there are two words "geber" and "öl" both of them  means "die", we can use "öl" and "geber" for animals, and can not use "geber" for people. And in Turkish "geber" is insolent.

sorry for giving such not nice example!